RestaurantReservation.Db/RestaurantReservationDbContext.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace RestaurantReservation.Db
+{
+    public class RestaurantReservationDbContext:DbContext
+    {
+        public DbSet<Customers> Customers { get; set; }
+        public DbSet<Employees> Employees { get; set; }
+        public DbSet<MenuItem> MenuItems { get; set; }
+        public DbSet<OrderItems> OrderItems { get; set; }
+        public DbSet<Orders> Orders { get; set; }
+        public DbSet<Reservations> Reservations { get; set; }
+        public DbSet<Restaurants> Restaurants { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(
+                "Server=DESKTOP-P1OAHG7\\MSSQLSERVER06;Database=RestaurantReservationCore;Integrated Security=True;TrustServerCertificate=True");
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Customers>().HasKey(c => c.CustomersId);
```

Comment:
Of course, navigation properties implicitly defines the relationships (1-1, 1-m1 n-m) btw entities, but we need to have this done manually using ef core, can u search how that could be configured and done in ef core and apply it?